docs/vc_patch.md

```diff
@@ -0,0 +1,113 @@
+
+**Commands** are contained in "`{`" braces "`}`". They are not output themselves, but may produce their own output when interpreted.
+
+Commands are interpreted with a series of arguments, separated by whitespace (spaces, tabs, or newlines). Leading and trailing whitespace is ignored; for example, "`{  hex  @  4  }`" is interpreted the same as "`{hex @ 4}`".
+
+**Arguments** evaluate to numeric values. They may be any of the following:
+
+- Literal numbers in decimal (base 10, e.g. "`42`"), hexadecimal (base 16, e.g. "`0x2a`"), or octal (base 8, e.g. "`052`"). They may start with a plus sign "`+`". Numbers may not be negative.
+- Comparison operators: "`==`" is 0, "`>`" is 1, "`<`" is 2, "`>=`" is 3, "`<=`" is 4, "`!=`" is 5, and "`||`" is 0x11.
+- Symbol names from the two `.sym` files provided to `make_patch` may evaluate as their bank-relative address, or their absolute offset in the ROM, depending on the command. They may also be followed by a plus sign and a literal number that gets added to the value.
+- "`@`" evaluates as the address or absolute offset of the current patch label.
+
+Any other characters are output as-is.
+
+
+## Patch template commands
+
+### `patch`, `PATCH`, `patch_`, or `PATCH_`
+
+Seeks the patched ROM contents between the current patch label, and the label which is the current patch label plus "`_End`". Outputs the bytes between those labels as a hexadecimal number "<code>0x<i>V</i></code>" for only one byte, or as a *value series* "<code>a<i>N</i>: <i>V1</i> <i>V2</i> [...] <i>VN</i></code>" for multiple bytes.
+
+An optional argument is an *offset* to add to the current patch label before gathering the contents between it and the end label.
+
+If the command name is all lowercase, the byte values use lowercase for hexadecimal digits A-F; if it is all uppercase, they use uppercase. If the command name ends in an underscore, a space is output after the colon preceding the values; if not, then it is not.
+
+For example, if "`{patch}`" outputs "`a3:ab cd ef`", then "`{PATCH_ 1}`" outputs "`a2: CD EF`", and "`{patch 2}`" outputs "`0xef`".
+
+Converting the patch template will print a warning if any differences exist between the original and patched ROMs, which are not covered by "`patch`" commands.
+
+
+### `dws`, `DWS`, `dws_`, or `DWS_`
+
+Outputs the alternating low and high bytes of its arguments as a value series.
+
+Symbol names or "`@`" are evaluated as their relative address.
+
+If the command name is all lowercase, the byte values use lowercase; if it is all uppercase, they use uppercase. If the command name ends in an underscore, a space is output after the colon; if not, then it is not.
+
+For example, if "`{dws_ 42 0xabcd wCurSpecies}`" outputs "`a6: 2a 00 cd ab 60 cf`", then "`{DWS >= wCurSpecies+3}`" outputs "`a4:04 00 63 cf`".
+
+
+### `db`, `DB`, `db_`, or `DB_`
+
+Outputs its argument as a single-byte value series.
```

Comment:
Outputs its arguments as a hexadecimal string of single-byte values.

Reply:
Maybe not the best proposal but "hexadecimal" and "single-byte"/"8 bit" are keywords here.